drivers/clock_control/clock_control_nrf_auxpll.c

```diff
@@ -113,7 +112,6 @@ static int clock_control_nrf_auxpll_init(const struct device *dev)
 	nrf_auxpll_ctrl_frequency_set(config->auxpll, config->frequency);
 
 	nrf_auxpll_lock(config->auxpll);
-	nrf_auxpll_trim_ctune_set(config->auxpll, sys_read8(config->ficr_ctune));
```

Comment:
For peripherals that can be trimmed in `init()`, trimming should be kept in separate drivers.